README.md

```diff
@@ -1,27 +1,23 @@
-# Docker-Mapserver
+<!-- 
+SPDX-FileCopyrightText: German Aerospace Center (DLR) <cosmoscout@dlr.de>
+SPDX-License-Identifier: CC0-1.0
+ -->
+# Map-Server Container for CosmoScout VR
 
-A dockerized mapserver for CosmoScout [csp-lod bodies](https://github.com/cosmoscout/cosmoscout-vr/tree/main/plugins/csp-lod-bodies#readme). The repo contains two dockerfiles `base.Dockerfile` and `example.Dockerfile`.
+A dockerized mapserver for CosmoScout [csp-lod bodies](https://github.com/cosmoscout/cosmoscout-vr/tree/main/plugins/csp-lod-bodies#readme). This repo has been created taking three use-cases into consideration - `loading example map data`, `loading custom map data` and `creating custom map data containers`. Each of the use-cases are discussed briefly below.
 
-## base.Dockerfile
+## 1) Loading Example Map Data
+Here you run a dockerized mapserver using NASA's `Blue Marble` and the `Natural Earth image` datasets and `ETOPO1` elevation data.
```

Comment:
Would be cool if you could turn the dataset names into links to websites containing more information on the data.